viz_test.go

```diff
@@ -0,0 +1,250 @@
+package dataset
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/ipfs/go-datastore"
+	"io/ioutil"
+	"testing"
+)
+
+var Viz1 = &Viz{
```

Comment:
Unexported, don't use a capital: rename these three to testViz1, testViz2, testViz3.